src/pages/governance/_components/ConfirmVoteDialog.tsx

```diff
@@ -221,12 +221,27 @@ function VoteForProposal({
         </span>
       </div>
 
-      <span
-        data-testid="OnChainGovernance.VotingFlow.VoteForProposal.Masternode"
-        className="text-gray-600 dark:text-dark-gray-600 font-semibold text-sm"
-      >
-        Masternode
-      </span>
+      <div className="flex">
+        <span
+          data-testid="OnChainGovernance.VotingFlow.VoteForProposal.Masternode"
+          className="text-gray-600 dark:text-dark-gray-600 font-semibold text-sm"
+        >
+          Masternode
+        </span>
+        <InfoHoverPopover
+          className="ml-1 self-center"
+          description={
+            <div className="px-4 py-3 font-normal text-sm bg-white text-left text-gray-900 rounded-lg border border-gray-100 shadow-md max-w-xs dark:bg-gray-800 dark:border-gray-700 dark:text-dark-gray-900">
+              Masternode ID can be retrieved by using the command
+              <span className="font-medium bg-[#FFF5EA] dark:bg-[#FF9F0A40]/[0.25]  mx-1 px-1 py-0.5 rounded">
```

Comment:
check tailwind config, we have these two custom color
```suggestion
              <span className="font-medium bg-[#FFF5EA] dark:bg-[#FF9F0A40]/[0.25] mx-1 px-1 py-0.5 rounded">
```